checks/evaluation/pinned_dependencies.go

```diff
@@ -260,6 +267,15 @@ func createReturnForIsPipInstallPinned(pr map[checker.DependencyUseType]pinnedRe
 		dl)
 }
 
+// Create the result for npm install commands.
+func createReturnForIsNpmInstallPinned(pr map[checker.DependencyUseType]pinnedResult,
+	dl checker.DetailLogger,
+) (int, error) {
+	return createReturnValues(pr, checker.DependencyUseTypeNpmCommand,
```

Comment:
i poked through the CL per your request. nothing stands out, but here and elsewhere: the google style guide would suggest a single line function invocation, rather than putting params on other lines

(though, I can see what this codebase does not generally do that, so your call on whether you want to start now)

https://google.github.io/styleguide/go/decisions#function-formatting

everything else looks fine (there's not too much code, so easy to review!)

Reply:
Thanks for reviewing, Jean!

> I can see what this codebase does not generally do that, so your call on whether you want to start now

Yeah, I will make the call to not address this in this PR. But will suggest this change for this file in a new PR! I do see the benefits.